### PR TITLE
disable strict status check (set it to loose)

### PR DESCRIPTION
**Pull request by wkloucek**
## Description
The function `This branch is out-of-date with the base branch` makes it very hard to merge external contributions (PRs merging from forks) during busy days. I'm not really sure if we need this feature since Drone auto-merges the PR branch to master anyways if there are no merge conflicts. 

The main problem is that we can not update the PR branch on forks and people might not invest enough time to get it merged.

## Motivation and Context
This PR changes the required status check from strict to loose (https://docs.github.com/en/github/administering-a-repository/about-protected-branches). How to do this in the YAML config can be seen here https://registry.terraform.io/providers/integrations/github/latest/docs/resources/branch_protection#required-status-checks

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [ ] Tests only (no source changes)

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link>

### CODE REVIEW

**Review by kulmann — APPROVED** [review]
I'm fine with this. Please wait for @micbar's review before merging.

**Comment by micbar** [comment]
lets give it a try.

Not sure if it does the trick. If not, i will change in UI

**Comment by micbar** [comment]
no wait, this is wrong.

**Review by micbar — CHANGES_REQUESTED** [review]
different setting

**Comment by micbar** [comment]
Screenshot_2021-01-29 owncloud ocis

You want the second one unchecked, correct?

**Comment by wkloucek** [comment]
> different setting

should be the right setting. This does not disable it completely. It just unticks "Require branches to be up to date before merging"

image

**Comment by wkloucek** [comment]
> Screenshot_2021-01-29 owncloud ocis
> 
> You want the second one unchecked, correct?

correct

**Comment by micbar** [comment]
IMO the naming is not the best :-)
we can give it a try, not sure if it is the right parameter in settings.yml

**Review by micbar — APPROVED** [review]

File: .github/settings.yml
Comment (phil-davis) [inline .github/settings.yml:42]:
```suggestion
        loose: true
```
I guess it might be something like this to force CI to be green, but not force the branch to be up-to-date with master.
But I am struggling to find examples.

**Comment by wkloucek** [comment]
@micbar @phil-davis looks good so far!?

**Comment by phil-davis** [comment]
> @micbar @phil-davis looks good so far!?

yes, it still says "Required statuses must pass before merging" while CI is running, which is good.

**Pull request merged**
